WordPress/gutenberg
# [RNMobile] Retrieve translation of color settings strings

**Gutenberg Mobile PR:** https://github.com/wordpress-mobile/gutenberg-mobile/pull/4479





## Description


The following strings located in `packages/components/src/mobile/color-settings/utils.native.js` file were not translated because they weren't retrieving their translation texts:
- `Linear`
- `Radial`
- `Solid`
- `Gradient`

This PR adds the `__` function to those strings for getting their translations.

## How has this been tested?



1. Change the device's locale to non-English.
2. Open a post/page.
3. Add a Cover block and select an image.
4. Select the Cover block.
5. Open the block settings by tapping on the gear icon button.
6. Tap on the Color option.
7. Observe that both tab strings "Solid" and "Gradient" are translated.
8. Tap on the "Gradient" tab.
9. Select any gradient color.
10. Tap on "Customize Gradient".
11. Observe that strings "Linear" and "Radial" are translated.

## Screenshots 

|    |    |
| --- | --- |
| <img src=https://user-images.githubusercontent.com/14905380/149771357-4e37f7ca-5ee0-4430-a1e9-214e8b3e459c.png> | <img src=https://user-images.githubusercontent.com/14905380/149771367-4a5bef49-555e-4f98-bb70-4c46f5f6f212.png> |

## Types of changes




Bug fix

## Checklist:
